impeller/entity/entity_pass.cc

```diff
@@ -1023,6 +1023,16 @@ std::unique_ptr<EntityPass> EntityPass::Clone() const {
 
   auto pass = std::make_unique<EntityPass>();
   pass->SetElements(std::move(new_elements));
+  pass->backdrop_filter_reads_from_pass_texture_ =
+      backdrop_filter_reads_from_pass_texture_;
+  pass->advanced_blend_reads_from_pass_texture_ =
+      advanced_blend_reads_from_pass_texture_;
```

Comment:
(These read-counts are what's important for `DrawPicture`.)

Reply:
You need the readcounts, the delegate, and the backdrop proc or wonderous will crash.

Reply:
That shouldn't be the case if the cloned pass is being used for `DrawPicture` (the elements are copied inline and these values should get discarded).

Reply:
For the delegate and backdrop proc, that is. The read counts are important.

Reply:
I can confirm that wonderous does not work correctly without copying the backdrop prop and entity delegate. Maybe the entity delegate _doesn't_ need to be copied but it does need to be set to a non-default value.

Let me double check that.

Reply:
Ah nevermind, we don't need these for the inline merge, but we definitely need to copy them for the recursive clone, otherwise Very Bad Things will happen: https://github.com/flutter/engine/blob/494fd7fe85a30ce4dfc28058961bb71920f514cc/impeller/entity/entity_pass.cc#L1015